modules/ROOT/pages/ext-analytics-elk.adoc

```diff
@@ -0,0 +1,174 @@
+= Analyze Business and API Data Using ELK
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Use the Elastic Stack (ELK stack) to analyze the business data and API analytics generated by Anypoint Platform Private Cloud Edition (Anypoint Platform PCE). You can use Filebeat to process Anypoint Platform log files, insert them into an Elasticsearch database, and then analyze them with Kibana.
+
+The following example shows how API data appears in Kibana:
+
+image::kibana-example.png[]
+
+== Prerequisites
+
+* Required software
++
+To export external analytics data to ELK, you must have the following software:
++
+** Mule runtime engine (Mule), version 3.8.4 or later
+** Anypoint Runtime Manager agent, version 1.7.0 or later
+** Elasticsearch, version 7.6.2
+** Filebeat, version 7.6.2
+** Kibana, version 7.6.2
+
+* Ensure that you have installed and configured your Mule runtime engine instances.
+* Ensure that you have registered your Mule runtime engine instances with Runtime Manager.
+
+== Edit the wrapper.conf Properties File
+
+In your Mule runtime installation, edit the following properties in the `conf/wrapper.conf` file:
+
+. Set the `analytics_enabled` property to `true`:
++
+----
+wrapper.java.additional.<n>=-Danypoint.platform.analytics_enabled=true
+----
+
+. Remove the URI value from the `analytics_base_uri` property:
++
+----
+wrapper.java.additional.<n>=-Danypoint.platform.analytics_base_uri=
+----
+
+. Set the `on_prem` property to `true`:
++
+----
+wrapper.java.additional.<n>=-Danypoint.platform.on_prem=true
+----
+
+== Enable Event Tracking
+
+To export business and event tracking data, you must enable event tracking in Runtime Manager:
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Select *Servers*.
+. Select the row containing the server on which you want to configure ELK.
+. Select *Manager Server* and then select the *Plugins* tab.
+. In *Event Tracking*, select *Business Events* from the *Levels* drop-down menu.
+. Select the switch to enable event tracking on ELK.
++
+Runtime Manager displays the *ELK Integration* dialog.
+
+. Configure the log files as necessary.
+. Select *Apply*.
+
+== Enable API Analytics
+
+To export API analytics, you must enable API analytics in Runtime Manager.
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Select *Servers*.
+. Select the row containing the server on which you want to configure ELK.
+. Select the switch to enable API analytics and then select the switch next to *ELK*.
+. Configure the log files as necessary.
+. Select *Apply*.
+
+== Install and Configure the Filebeat Agent
+
+On each server with Mule runtime instances, download and install Filebeat for your operating system.
+
+== Configure the Mule Filebeat Module
+
+On each server with Mule runtime instances, download and expand the Filebeat Module archive from the following URL:
++
```

Comment:
Does this render ok in docs?
![image](https://user-images.githubusercontent.com/473061/84198212-0d960180-aa58-11ea-9e48-7e6b03b00e99.png)